woocommerce/woocommerce-gutenberg-products-block
# Create GitHub actions for automated tests

Fixes #3542 and incorporates @haszari 's PHP test workflow found in #3584

### How to test the changes in this Pull Request:

1. Push a commit to this branch that breaks one or more of the PHP tests.
2. Verify the "PHP, JavaScript, and E2E tests" workflow fails.
3. Revert the commit and push a new commit to the branch that breaks one of the JS unit tests.
4. Verify the "PHP, JavaScript, and E2E tests" workflow fails.
5. Revert the commit and push a new commit to the branch that breaks one of the JS E2E tests.
6. Verify the "PHP, JavaScript, and E2E tests" workflow fails.
7. Revert the commit and leave the branch as you found it.
